frontend/lib/util/format.test.ts

```diff
@@ -16,4 +18,17 @@ describe("Format util", () => {
     expect(formatNumber(input)).equals(expected);
     expect(deformatNumber(expected)).equals(parseInt(input, 10));
   });
+
+  const today = new Date(new Date().setHours(10, 20));
+  let yesterday = new Date(today);
+  yesterday = new Date(yesterday.setDate(today.getDate() - 1));
+  let one_week_ago = new Date(today);
+  one_week_ago = new Date(one_week_ago.setDate(today.getDate() - 7));
```

Comment:
```suggestion
  let today = new Date();
  today.setHours(10, 20);
  let yesterday = new Date();
  yesterday.setDate(today.getDate() - 1);
  let one_week_ago = new Date();
  one_week_ago.setDate(today.getDate() - 7);
```

Reply:
Thanks! Changed in a586e6d004d797346ff9bfea130f5901aa6593bc